react/states_and_effects/project_cv_application.md

```diff
@@ -24,6 +24,10 @@ While working on this project, you might notice that some parts of your code exe
 1. Include a `styles` directory under your `src` directory for your CSS files. You'll need to import these in the component files to use them.
 1. Push the results and deploy them with any of the options mentioned below. At this point of the curriculum, it doesn't matter which platform you choose as long as your project is live on the internet!
 
+#### Extra credit
+
+Allow your users to download the CV as a pdf file. You can use [react-to-pdf](https://www.npmjs.com/package/react-to-pdf) library to achieve this.
+
```

Comment:
The CV app is learner's first project in React, and I'm expecting the project to be a little challenging for them already. I don't think we should include this extra credit as the goal is to get learners familiar with React with these earlier CV and memory card projects. This but an unneeded complexity IMO. What do you think?

I'd also be curious what the @TheOdinProject/react team thinks

Reply:
I'm with you on keeping the first project intentionally narrow in scope. If someone wants to expand the functionality and wants advice on that, I'd rather encourage them to keep the narrow focus and remember the point of the project, as opposed to expand beyond its purpose.